index.html

```diff
@@ -4,17 +4,289 @@
+                                <li class="nav-item">
+                                    <a class="nav-link active" aria-current="page" href="#"> Home</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="./pages/about.html">About</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="./pages/services.html">Services</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="./pages/blog.html">Blog</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="./pages/contact.html">Contact</a>
+                                </li>
+                            </ul>
+                    
+                        <h1 class="padding-top col-md-5 offset-md-1 align-self-baseline">Maria Hope, Ph.D. - Psychologist</h1>
+                    </div>
+                </div>
+                
+            
+        </nav> 
+    
+    </header>
+    <main class="text-center container">
+        <section class="hero row">
+            <div class="hero-md-img p-0 display-order col-md-6">
+            </div>
+                <img class="hero-img p-0 col-12 display-sm img-fluid" src="./img/erol-ahmed-aIYFR0vbADk-unsplash.jpg" alt="Succulent plant">
+                <section class="hero-text box-sizing col-10 offset-1 col-md-6 offset-md-0 aliign align-self-end ">
+                    <h2>Online Therapy for Emotional Health</h2>
+                    <p>Lorem ipsum dolor sit amet consectetur. </p> 
+                </section>
+        </section>
+        <section class="about row">
+            <article class="about-text box-sizing">
+                <div class="col-10 offset-1 col-md-8 offset-md-2" >
+                    <h3>about</h3>
+                    <h2>Maria Hope</h2>
+                    <p> Lorem ipsum dolor sit amet consectetur adipisicing 
+                        elit. Dicta perspiciatis eos, facilis maiores nam consectetur libero 
+                        perferendis, saepe soluta culpa voluptatum minus animi eveniet earum. 
+                        Neque reprehenderit sed dolore omnis?Praesentium fuga consequatur 
+                        vitae ut ex et maiores quisquam id voluptates dolorem!</p> 
+                </div>
+                
+            </article>
+            <img class="col-10 offset-1 box-sizing align-self-center" src="./img/pexels-christina-morillo-1181690.jpg" alt="Image of Maria Hope">
+        </section>
+        <section class="therapy-sect row"> 
+            <article class="box-sizing box-bg-1 indiv-therapy col-12 col-md-12 ">
```

Comment:
Your text isn't quite vertically aligned. Use `display: flex` with `align-items: center` and equalise your top and bottom padding to help with this.